### PR TITLE
Prometheus: Fix range variables interpolation for time ranges smaller than 1 second

### LINKED ISSUE #42123: prometheus: time ranges smaller than 1 second are interpolated in variables incorrectly
the variables `$__range` , `$__range_s` and `$__range_ms` do not behave the same in grafana8.3.0-beta1 as they did in grafana8.2.5, when the selected time-range is smaller than 1 second.

you can reach such time-ranges by zooming in the graph with the mouse for example.

examples:
1.
- your time-range is 376milliseconds.
- you have the grafana query `go_goroutines{instance!="$__range $__range_s $__range_ms"}`
- in grafana8.2.5 this becomes: `go_goroutines{instance!="0s 0 376"}`
- in grafana8.3.0-beta1 this becomes: `go_goroutines{instance!="0s 0 0"}`

2.
- your time-range is 800milliseconds.
- you have the grafana query `go_goroutines{instance!="$__range $__range_s $__range_ms"}`
- in grafana8.2.5 this becomes: `go_goroutines{instance!="1s 1 800"}`
- in grafana8.3.0-beta1 this becomes: `go_goroutines{instance!="0s 0 0"}`

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
This PR fixes range variables interpolation for time ranges smaller than 1 second. You can review https://github.com/grafana/grafana/issues/42123 for more details and how to reproduce the issue. 

I am also following approach "stolen" during reviewing @gabor PR https://github.com/grafana/grafana/pull/42126 🙈 

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana/issues/42123

### CODE REVIEW

File: pkg/tsdb/prometheus/time_series_query.go [hunk 1]
```diff
@@ -262,6 +254,20 @@ func calculateRateInterval(interval time.Duration, scrapeInterval string, interv
 	return rateInterval
 }
 
+func interpolateVariables(expr string, interval time.Duration, timeRange time.Duration, intervalCalculator intervalv2.Calculator, timeInterval string) string {
+	rangeMs := timeRange.Milliseconds()
+	rangeSRounded := int64(math.Round(float64(rangeMs) / 1000.0))
+	fmt.Println(rangeMs, rangeSRounded)
```
Comment (gabor) [inline pkg/tsdb/prometheus/time_series_query.go:260]: you probably want to remove this `fmt.Println`...

**Review by gabor — APPROVED** [review]
LGTM! works fine, there is a probably-forgotten debug-print-statement i mentioned in my previous comment, except that it's all ok 👍

**Review by joey-grafana — APPROVED** [review]

File: pkg/tsdb/prometheus/time_series_query.go (hunk 1 above)
Comment (ivanahuckova) [reply pkg/tsdb/prometheus/time_series_query.go:260]:
```suggestion
```

Comment (ivanahuckova) [reply pkg/tsdb/prometheus/time_series_query.go]: Yes!  Good catch. Removed

**Pull request merged**